### PR TITLE
Setting scans retention period to zero, so scans can be deleted

### PR DESCRIPTION
**Pull request by Bischoff**
Solves testsuite failure when removing openscap scans

### CODE REVIEW

File: features/salt_openscap_audit.feature [hunk 1]
```diff
@@ -47,9 +48,16 @@ Feature: Use the openSCAP audit feature in SUSE Manager for a salt minion
...
+  Scenario: Remove audit scans retention period
+    Given I am on the Organizations page
+    And I follow "SUSE Test" in the content area
+    And I follow "Configuration" in the content area
+    When I enter "0" as "scap_retention_period"
+    And I click on "Update Organization"
+    Then I should see a "Organization SUSE Test was successfully updated." text
+
+  Scenario: Delete audit results
+    Given I am on the Systems overview page of this "sle-minion"
```
Comment (MalloZup) [inline features/salt_openscap_audit.feature]: Add cleanup scenario to set the var to default

**Comment by Bischoff** [comment]
True. Doing it now.

**Comment by MalloZup** [comment]
@Bischoff looks good.

Can you rename this Scenario name to be more precise?

https://github.com/Bischoff/spacewalk-testsuite-base/blob/2a27e87a2f9dbbb77ea4d2c7ebcdd3406961c6b9/features/salt_openscap_audit.feature#L30

What is the intention/goal to have an quite identical audit?

**Comment by Bischoff** [comment]
I'll merge the pull request because I need this in the test suite review for tomorrow.

About the name: I don't see how this is unprecise. Maybe the fact that it is "almost identical" is unrelevant and could be removed. It is almost identical because only one IPv6 variable change is detected.

**Comment by MalloZup** [comment]
It is not clear. Why you make a second run identical then?

**Pull request merged**